package.json

```diff
@@ -83,6 +83,7 @@
     "eslint-plugin-tailwindcss": "npm:@hasparus/eslint-plugin-tailwindcss@3.17.5",
     "fs-extra": "11.2.0",
     "graphql": "16.9.0",
+    "gray-matter": "^4.0.3",
```

Comment:
We import it in `generate-changelog.js` and previously it allowed us to grab a transitive dep from Nextra, but now we need to install it.